mat64/eigen.go

```diff
@@ -817,3 +820,49 @@ func (f EigenFactors) D() *Dense {
 	}
 	return dm
 }
+
+// MarshalBinary encodes the receiver into a binary form and returns the result.
+func (f EigenFactors) MarshalBinary() ([]byte, error) {
```

Comment:
my rationale for adding them was that they might be intermediate steps of an analysis and one might want to restart from these intermediate states w/o having to go through the whole (possibly quite expensive) computation.
as a user, it might become a bit tedious to figure out that some critical fields are not exported and thus silently discarded from the compiler-provided default serialization functions.
as a library implementer, providing those seemed quite interesting.

but, ok. I can remove them, if the general consensus tips that way. :)